### ISSUE TITLE
2.5.0 Can't sync with AGP 8.2.0

### ISSUE THREAD

**Issue by DorBSoft** [LikeTheSalad/android-stem#48]
After upgrading AGP to 8.2.0, which upgrades Gradle to 8.2.
```
An exception occurred applying plugin request [id: 'com.likethesalad.stem']
> Failed to apply plugin class 'com.likethesalad.tools.android.plugin.AndroidToolsPlugin'.
   > com.android.build.api.variant.AndroidComponentsExtension.getPluginVersion()
```

If I downgrade to android-stem release 2.4.1, then it syncs fine.

**Comment by CanerKaraca23** [comment]
I have this issue too @LikeTheSalad

**Comment by LikeTheSalad** [comment]
Thanks for your feedback! I think I know what's going on, I'll take a look at it later today.

**Comment by LikeTheSalad** [comment]
This should be fixed in the new version `2.6.0`, thanks for the feedback.
